requirements/base.txt

```diff
@@ -109,9 +109,7 @@ meilisearch==0.34.0
 mysqlclient==2.2.7
     # via -r requirements/base.in
 newrelic==10.7.0
```

Comment:
Looks like newrelic is still included in the requirements, is that intentional?

Reply:
Yes -- if you look at the comments below that line, you can see that it's still being pulled in via edx-django-utils (for now).

Reply:
What's the future plan for removing it from requirements?

Reply:
As far as I can tell, this was the last instance in the edx and openedx orgs where we still had a direct `import newrelic` -- so after this merges, I think we're clear to remove `newrelic` from the edx-django-utils `base.in` as well. I don't have a specific plan for that, but I did want to clear the path for it.

Reply:
Update: Filed a DEPR for this: https://github.com/openedx/public-engineering/issues/360